Formula/gcc@5.rb

```diff
@@ -64,15 +64,17 @@ class GccAT5 < Formula
     end
   end
 
+  def version_suffix
+    version.major.to_s
+  end
```

Comment:
Why does this need to move?

Reply:
It's used in the postinstall block for Linux. I forgot to move it while backporting the code.

Reply:
But why not just do the version.major

Reply:
I think this comes from the main gcc formula where it can be HEAD. So all formulae are aligned and have this extra function. But we could also align all the gcc@x formulae to use just version.major. I would just leave it as-is.